File: SavaariAPI/Driver.py
"""
	Savaari Application Programming Interface
	Nabeel Danish               18I-0579
	Muhammad Farjad Ilyas       18I-0436
"""
# =========================================================================
# 						 		DRIVER CLASS
# =========================================================================

from User import *
from Location import *

# Nabeel Danish
class Driver(User):

	# Constructors
	def __init__(self, *args, **kwargs):
		super(Driver, self).__init__(*args, **kwargs)

		# Class attributes
		self.__last_location = kwargs.get('last_location')
		self.__activeStatus = kwargs.get('')

	# End of __init__

	# Getters and Setters
	def getLastLocation(self):
		return self.__last_location

	def setLastLocation(self, last_location):
		self.__last_location = last_location

# End of class: Driver

File: savapi/demo/src/main/java/com/savaari_demo/Driver.java
package com.savaari_demo;

public class Driver extends User 
{
	// Main Attributes
	Boolean isActive;
	Boolean isTakingRide;
	
	// Main Constructors
	public Driver() {
		super();
	}
	public Driver(Location lastLocation, Boolean isActive, Boolean isTakingRide) {
		super();
		this.lastLocation = lastLocation;
		this.isActive = isActive;
		this.isTakingRide = isTakingRide;
	}
	
	// Getters and Setters
	public Boolean getIsActive() {
		return isActive;
	}
	public void setIsActive(Boolean isActive) {
		this.isActive = isActive;
	}
	public Boolean getIsTakingRide() {
		return isTakingRide;
	}
	public void setIsTakingRide(Boolean isTakingRide) {
		this.isTakingRide = isTakingRide;
	}
}
